test/e2e/framework/util.go

```diff
@@ -328,7 +328,10 @@ func waitForServiceAccountInNamespace(c clientset.Interface, ns, serviceAccountN
 		}
 		return false, nil
 	})
-	return err
+	if err != nil {
+		return fmt.Errorf("wait for service account %q in namespace %q: %v", serviceAccountName, ns, err)
```

Comment:
do you want to be pedantic and use %w, :smile:

Reply:
I don't think there is a guidance to always use %w, is there?

It boils down to "does it make sense here". The caller is not going to check for specific errors, so that reason doesn't apply here. We loose some details because ExpectNoError will dump the entire wrapped error - that may be a reason to use %w. Okay, let me change that.

Reply:
agree, it was just a comment to get your opinion :)